commands.c

```diff
@@ -81,14 +82,69 @@ command_answer *send_command(char *name, char *data, task_context *context)
     return cmd_answer;
 }
 
+command_answer *send_attest_command(direction direction, struct sock *s, u16 port)
+{
+    int iplen = 16;
+    char ipformat[] = "%pI4";
+
+    if (s->sk_family == AF_INET6)
+    {
+        iplen = 46;
+        strcpy(ipformat, "%pI6");
+    }
```

Comment:
I would avoid the strcpy function, instead I would use `cont char* ipformat`. and simply update the pointer value inside the if.